eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/consumer/SubStreamHandler.java

```diff
@@ -47,13 +48,11 @@ public class SubStreamHandler<T> extends Thread implements Serializable {
     private final transient ConsumerServiceStub consumerAsyncClient;
 
     private final transient EventMeshGrpcClientConfig clientConfig;
-
-    private transient StreamObserver<CloudEvent> sender;
-
     private final ReceiveMsgHook<T> listener;
+    private transient StreamObserver<CloudEvent> sender;
 
     public SubStreamHandler(final ConsumerServiceStub consumerAsyncClient, final EventMeshGrpcClientConfig clientConfig,
-        final ReceiveMsgHook<T> listener) {
+                            final ReceiveMsgHook<T> listener) {
         this.consumerAsyncClient = consumerAsyncClient;
```

Comment:
Redundant indention and field sequence problem here. 

**I will refrain from further annotating similar questions below. You can independently review them on this page:**

![image](https://github.com/apache/eventmesh/assets/41445332/0ad19437-0597-4923-adf4-d7c26d030ff7)